lib/form1010_ezr/service.rb

```diff
@@ -60,24 +64,22 @@ def submit_form(parsed_form)
       log_and_raise_error(e, parsed_form)
     end
 
-    def log_submission_failure(parsed_form)
-      StatsD.increment("#{Form1010Ezr::Service::STATSD_KEY_PREFIX}.failed_wont_retry")
+    def log_submission_failure(parsed_form, retries_exhausted: false)
```

Comment:
Adding this boolean will allow us to see if the failure was the last retry attempt or not.